youtube_dl/extractor/bilibili.py

```diff
@@ -82,10 +82,10 @@ def _real_extract(self, url):
                 'formats': formats,
             })
 
-        title = self._html_search_regex('<h1[^>]+title="([^"]+)">', webpage, 'title')
+        title = self._html_search_regex(r'<h1[^>]+title="([^"]+)">', webpage, 'title')
         description = self._html_search_meta('description', webpage)
         timestamp = unified_timestamp(self._html_search_regex(
-            r'<time[^>]+datetime="([^"]+)"', webpage, 'upload time', fatal=False))
+            '<time[^>]+datetime="([^"]+)"', webpage, 'upload time', fatal=False))
```

Comment:
Thanks for brining up this. It's indeed a mistake.

First I replace all ```(?s)_search_regex\((\s*)'``` with ```_search_regex(\1r'``` and revert occurrences without invalid escape sequences. This ```r``` is accidentally removed during the revert phase.

Now a question is: should youtube-dl use raw strings for all regular expressions, or only for those that trigger errors on 3.6?